front/pages/poke/templates/[tId].tsx

```diff
@@ -226,6 +254,107 @@ function TextareaField({
   );
 }
 
+function PresetActionsField({
+  control,
+  name,
+  title,
+}: {
+  control: Control<CreateTemplateFormType>;
+  name: keyof CreateTemplateFormType;
+  title?: string;
+}) {
+  const { fields, append, remove } = useFieldArray({
+    control,
+    name: "presetActions",
+  });
+
+  return (
+    <PokeFormField
+      control={control}
+      name={name}
+      render={() => {
+        return (
+          <PokeFormItem>
+            <PokeFormLabel className="capitalize">
+              {title ?? name}
+            </PokeFormLabel>
+            {fields.map((field, index) => (
+              <div key={field.id} className="grid grid-cols-5 gap-4">
+                <InputField
+                  control={control}
+                  // @ts-expect-error - TS doesn't like the dynamic key
```

Comment:
Weird 🤨

Reply:
That's cause we say name is `keyof CreateTemplateFormType;` but I did not found an easy way to say -> look inside this type this key is also valid